assets/js/theme/catalog.js

```diff
@@ -3,6 +3,23 @@ import urlUtils from './common/utils/url-utils';
 import Url from 'url';
 
 export default class CatalogPage extends PageManager {
+    constructor(context) {
+        super(context);
+
+        window.addEventListener('beforeunload', () => {
+            if (document.activeElement.id === 'sort') {
+                window.localStorage.setItem('sortByStatus', 'selected');
+            }
+        });
+    }
+
+    arrangeFocusOnSortBy($elementSelector) {
```

Comment:
I believe we could skip passing selector here

Reply:
I've added it in case the sortBy Selector will be change later for some reason. Removed it.